iib/workers/tasks/utils.py

```diff
@@ -147,7 +154,7 @@ def set_registry_token(token, container_image):
 
 
 @retry(wait_on=IIBError, logger=log)
-def skopeo_inspect(*args, return_json=True):
+def skopeo_inspect_no_cache(*args, return_json=True):
```

Comment:
Why not simply keep this `skopeo_inspect` and have a boolean `use_cache` defaulting to `False`. You can probably use CacheRegion.get_or_create() if `use_cache` is `True`. I believe that would be a cleaner implementation. Thoughts?

Reply:
OK, I used just the single function and specify cache by parameter. Default to using cache.

Reply:
Code was updated and now it uses just one function.